spec/factories/fee_schemes.rb

```diff
@@ -25,28 +25,34 @@
 
     trait :agfs_twelve do
       start_date { Settings.clar_release_date.beginning_of_day }
-      end_date { Settings.agfs_scheme_13_clair_release_date.beginning_of_day - 1.day }
+      end_date { Settings.agfs_scheme_13_clair_release_date.end_of_day - 1.day }
       version { 12 }
     end
 
     trait :agfs_thirteen do
       start_date { Settings.agfs_scheme_13_clair_release_date.beginning_of_day }
-      end_date { Settings.agfs_scheme_14_section_twenty_eight.beginning_of_day - 1.day }
+      end_date { Settings.agfs_scheme_14_section_twenty_eight.end_of_day - 1.day }
       version { 13 }
     end
 
     trait :agfs_fourteen do
       start_date { Settings.agfs_scheme_14_section_twenty_eight.beginning_of_day }
-      end_date { Settings.agfs_scheme_15_additional_prep_fee_and_kc.beginning_of_day - 1.day }
```

Comment:
i notice we're going from beginning_of_day to end_of_day is this an issue which is in our live database or just tests?

Reply:
I don't think it's caused any issues, but it was something we noticed while working on this that could be an edge case that could in theory crop up as there was a bit of a gap with how it was previously written